selfdrive/camerad/cameras/camera_common.h

```diff
@@ -34,6 +34,7 @@
 #define LOG_CAMERA_ID_QCAMERA 3
 #define LOG_CAMERA_ID_MAX 4
 
+// TODO: remove these once all the internal tools are moved to vipc
```

Comment:
Are they used for focussing lenses in production?

Reply:
yes, lens focusing and sharpness tests